server/src/mzb_k8s_plugin.erl

```diff
@@ -67,43 +67,51 @@ create_cluster(PluginOpts, NumNodes, ClusterConfig) when is_integer(NumNodes), N
     end.
 
 destroy_cluster(ID) ->
-    R = delete_rc(ID),
-    lager:info("Destroyed RC: ~p, result: ~p", [ID, R]),
+    R = delete_deployment(ID),
+    lager:info("Destroyed Deployment: ~p, result: ~p", [ID, R]),
     {ok, _} = R,
     ok.
 
 
-%%%%%%%%%%%%%%%%%%%% 
+%%%%%%%%%%%%%%%%%%%%
 % Internal API
-%%%%%%%%%%%%%%%%%%%% 
+%%%%%%%%%%%%%%%%%%%%
 
 context_arg(undefined) -> [];
 context_arg(Context) -> ["--context", Context].
 
 namespace_arg(undefined) -> [];
 namespace_arg(Namespace) -> ["--namespace", Namespace].
 
-create_rc({Context, Namespace, BenchName}, Image, PodSpec, NumNodes) ->
+create_deployment({Context, Namespace, BenchName}, Image, PodSpec, NumNodes) ->
+    JsonFile = write_json_resource(BenchName),
+    {ok, Binary} = file:read_file(JsonFile),
```

Comment:
Could you return the binary from the function? We don't need this temporal file at all.

Reply:
And why we need the file at all? Why don't just do `jiffy:encode()`?